de.dlr.sc.virsat.model.extension.tests.edit.test/src/de/dlr/sc/virsat/model/dvlm/dmf/DmfResourceTest.java

```diff
@@ -659,8 +660,85 @@ protected void doExecute() {
 		TestCategoryReference beanTestCategoryAssignmentReference = new TestCategoryReference(referencingCa);
 		
 		assertEquals("Reference is set correctly", beanTestCategoryAssignment, beanTestCategoryAssignmentReference.getTestRefCategory());
+		
+		resSeiOther.unload();
+		resSei.unload();
+		dmfCategoryAssignmentReference.setName("vfdv");

```

Comment:
Why setting these strange names?

Reply:
Oh that's left over from when I was debuging it, in the dedicated test case for it I use proper names

Reply:
I will remove it from this test case.